apis/Google.Cloud.Firestore/Google.Cloud.Firestore.IntegrationTests/QueryTest.cs

```diff
@@ -468,6 +470,69 @@ public async Task Count_WithLimit()
             Assert.Equal(2, snapshotWithLimit.Count);
         }
 
+        [Fact]
+        public async Task SumTest()
+        {
+            CollectionReference collection = _fixture.StudentsCollection;
+            var snapshot = await collection.Aggregate(Sum("Level", "Sum_Of_Levels"), Sum("MathScore"), Sum("EnglishScore"), Sum("Name")).GetSnapshotAsync();
+            Assert.Equal(Students.Data.Sum(c => c.Level), snapshot.GetValue<long>("Sum_Of_Levels")); // Long value, Alias check 
+            Assert.Equal(Students.Data.Sum(c => c.MathScore), snapshot.GetValue<double>("Sum_MathScore")); // Double value check
+            Assert.Equal(Double.NaN, snapshot.GetValue<double>("Sum_EnglishScore"));  //NaN value check
+        }
+
+        [Fact]
+        public async Task AvgTest()
+        {
+            CollectionReference collection = _fixture.StudentsCollection;
+            var snapshot = await collection.Aggregate(Average("MathScore"), Average("Level", "Avg_Of_Level"), Average("EnglishScore"), Average("Name")).GetSnapshotAsync();            
+            Assert.Equal(Students.Data.Average(c => c.MathScore), snapshot.GetValue<double>("Avg_MathScore")); // Double value check
+            Assert.Equal(Students.Data.Average(c => c.Level), snapshot.GetValue<double>("Avg_Of_Level")); // Alias check
+            Assert.Equal(Double.NaN, snapshot.GetValue<double>("Avg_EnglishScore")); //NaN value check
+        }
+
+        [Fact]
+        public async Task SumWithLimit()
```

Comment:
Given that we've already got Count tests with a limit, I don't think we need it for Sum/Average. We're really testing the server at this point, rather than our code. Ditto for the filter - I'd at least prefer to see that in a unit test than an integration test.